package/MDAnalysis/converters/RDKit.py

```diff
@@ -262,9 +267,32 @@ class RDKitConverter(base.ConverterBase):
     cache since the arguments given are different. You can pass a
     ``cache=False`` argument to the converter to bypass the caching system.
 
+    The ``_MDAnalysis_index`` property of the resulting molecule corresponds
+    to the index of the specific :class:`~MDAnalysis.core.groups.AtomGroup`
+    that was converted, which may not always match the ``index`` property.
+
+    To get a better understanding of how the converter works under the hood,
+    please refer to the following RDKit UGM presentation:
+
+    * `Video (4:55 to 8:05) <https://youtu.be/5b5wYmK4URU>`__
+    * `Slides <https://github.com/rdkit/UGM_2020/blob/master/Presentations/C%C3%A9dricBouysset_From_RDKit_to_the_Universe.pdf>`__
+
+    Their are some molecules containing specific patterns that the converter
+    cannot currently tackle correctly. See
+    `Issue #3339 <https://github.com/MDAnalysis/mdanalysis/issues/3339>`__ for
+    more info.
 
     .. versionadded:: 2.0.0
 
+    .. versionchanged:: 2.2.0
+        See the full changelog for more details:
```

Comment:
Please don't reference the CHANGELOG. The entry should be self-contained and provide the most important points. I also don't think that item lists properly format. They seem to be typeset separately. Just write prose over multiple lines. 

I don't think that the SanitizationError fix requires a "versionchanged" entry — this sounds like a bug fix to me. Mostly the versionchanged tells users if they need to be aware of changes in usage or output. (Although, if in dount, provide rather more information than less ;-) ).